src/main/java/net/sf/jabref/model/ChainNode.java

```diff
@@ -126,6 +126,7 @@ protected void setParent(T parent) {
      *
      */
     public void removeChild() {
+        // NPE if this is ever called
         child.setParent(null);
```

Comment:
Oh, yes. The problem is actually passing `null` to a metod with an `Objects.requireNonNull` as the first call... So clearly that code was never executed.